### ISSUE TITLE
Pam's New Workout

### ISSUE THREAD

**Issue by boourns** [boourns/synthmodes#64]
https://busycircuits.com/alm017/
https://busycircuits.com/docs/alm017-manual.pdf

I think the cheat sheet should cover
basic navigation,  
Per Trigger Adjustment parameters 
CV assigning.
saving/loading settings

**Comment by benirose** [comment]
How do we go about asking ALM if we can use images in the manual?

**Comment by boourns** [comment]
Hi Ben,

I'll send them an email and let you know what they say.

Cheers
Tom

**Comment by benirose** [comment]
OK. We might want to recreate some of it for the smaller screen anyways, similar to how the Mother 32 sequencer pages are, but the graphics in the manual are very helpful and I figure it's good to ask these kinds of things.

**Comment by boourns** [comment]
Email sent, I'll let you know their response

**Comment by benirose** [comment]
For the Extended Output Parameters, we could do a page for each:
Extended Output Parameters
Wave
Level
Offset
Width
Phase
Delay
Delay Divisor (Dly)
Slop
EStep
ETrig
ERot
RSkip
Loop
Quant
Logic

Or we could bundle them up like this:
Waveform Modifiers
- Wave
- Level
- Offset
- Width
- Phase
- Delay
- Delay Divisor (Dly)

Rhythm and Randomization
- EStep
- ETrig
- ERot
- Slop
- RSkip
- Loop

Quantization
Logic

Any thoughts? The second option would be just 4 pages in the "Extended Output Parameters" section.

**Comment by boourns** [comment]
Amazingly, nobody has ever complained about the layout of a cheat sheet, and sometimes I've done it one way and sometimes the other way.  So if you feel strongly one way or another go for it.

To me, I think the second case makes more sense to me for this module, grouping them like that.  That fits more with how I used the module when I owned one.

**Comment by benirose** [comment]
Yeah, the second one makes more sense to me too, I think when information is sparse across many pages it can be a less useful tool. I should be able to start a draft on the General and Extended Output Parameters soon, but I was having a hard time following the manual on mutes, saves, and CV control so I'll work on those after I've received mine and played around with it a bit.

**Comment by benirose** [comment]
Going to try to circle back to this in the next few weeks, I've definitely had time to play around with my Pams and know it pretty intimately now!

**Comment by boourns** [comment]
That would be awesome, thank you :)

Never head back from ALM so please no reuse of their manual.

**Comment by dylhunn** [comment]
I noticed this is tagged "ready to go" -- is this expected in an upcoming release?

**Comment by boourns** [comment]
Sorry, "ready to go" means it's ready to be written.  Pam's has not yet been written.

New release of the app is coming out likely today though!

**Comment by benirose** [comment]
Sorry I let this one fall off my radar. I’ve had a lot of time with Pam’s since I first commented so I’ll try to commit some time to this in the next week or two.

**Comment by boourns** [comment]
@benirose that'd be amazing! thank you!
